package.json

```diff
@@ -6,9 +6,15 @@
   "scripts": {
     "prepublish": "npm run build",
     "build": "babel --out-dir dist src",
-    "test": "mocha test",
+    "start": "node demo/server.js",
+    "watch": "nodemon -w demo -x node demo/server.js",
+    "test": "yarn build && yarn lint && yarn test:unit && yarn test:integration",
+    "test:unit": "mocha test",
+    "test:integration": "start-server-and-test start http://localhost:3000 cy:run",
```

Comment:
What about using a different port than 3000? Then a developer could keep yarn watch running in a different terminal and still run integration tests.

Reply:
`yarn cy:run` wont work unless you have a server started.
if you have no server started you can run `yarn test:integration` (it will start and stop a server itself with `start-server-and-test` package)